lib/src/model/movie_result.dart

```diff
@@ -53,18 +53,21 @@ class MovieResult {
       voteCount: parsedJson['vote_count'],
       id: parsedJson['id'],
       video: parsedJson['video'],
-      voteAverage: parsedJson['vote_average'],
-      title: parsedJson['title'],
+      voteAverage:
+          parsedJson['vote_average'] != null ? parsedJson['vote_average'] : 0,
+      title: parsedJson['title'] != null ? parsedJson['title'] : 'No Name',
       popularity: parsedJson['popularity'],
       posterPath:
           parsedJson['poster_path'] != null ? parsedJson['poster_path'] : '',
       originalLanguage: parsedJson['original_language'],
       originalTitle: parsedJson['original_title'],
       genreIds: _getGenresIdList(genreIds),
       backdropPath: parsedJson['backdrop_path'],
-      adult: parsedJson['adult'],
-      overview: parsedJson['overview'],
-      releaseDate: parsedJson['release_date'],
+      adult: parsedJson['adult'] != null ? parsedJson['adult'] : false,
+      overview: parsedJson['overview'] != null ? parsedJson['overview'] : '',
```

Comment:
' ' can be added to String file like emptyString

Reply:
Done.. I prefered to create a distinct default constant for each value to be able to modify them independently